test/unit/media/headset.test.ts

```diff
@@ -167,6 +167,31 @@ describe('SdkHeadset', () => {
       expect(setHoldSpy).toHaveBeenCalledWith('123', false);
     });
   });
+
+  describe('listenForSessionEvents', () => {
+    it('should end currentCall on sessionEnded', () => {
+      sdkHeadset.endCurrentCall = jest.fn();
+
+      const conversationId = 'myConvoId';
+      sdk.emit('sessionEnded', { conversationId } as any, null);
+
+      expect(sdkHeadset.endCurrentCall).toHaveBeenCalledWith(conversationId);
+    });
+
+    it('should set ringing if not autoanswer', () => {
+      sdkHeadset.setRinging = jest.fn();
+
+      sdk.emit('pendingSession', { autoAnswer: false, conversationId: '15215asdf'} as any);
+      expect(sdkHeadset.setRinging).toHaveBeenCalled();
+    });
+
+    it('should not set ringing if autoanswer', () => {
+      sdkHeadset.setRinging = jest.fn();
+
+      sdk.emit('pendingSession', { autoAnswer: true, conversationId: '15215asdf'} as any);
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/MyPureCloud/genesys-cloud-webrtc-sdk/pullRequest?prid=9596369)
```suggestion
      sdk.emit('pendingSession', { autoAnswer: true, conversationId: '15215asdf'} as unknown);
```